docs/src/charm/howto/cos-lite.md

```diff
@@ -30,13 +35,15 @@ juju deploy ubuntu microk8s --series=focal --constraints="mem=8G cores=4 root-di
 ```
 
 Deploy MicroK8s on Ubuntu by accessing the unit you created at the last step
-with `juju ssh microk8s/0` and following the [Install Microk8s][how-to-install-microk8s]
-guide for configuration.
+with `juju ssh microk8s/0` and following the 
+[Install Microk8s][how-to-install-microk8s] guide for configuration.
```

Comment:
Why do we have microk8s in this doc?

Reply:
I guess because at the time it was written we didn't have a reliably working Canonical K8s. I will make a card to update it

Reply:
https://warthogs.atlassian.net/browse/KU-1145

Reply:
Thanks Nick!